pkg/common/helpers/clusters_test.go

```diff
@@ -1,114 +0,0 @@
-package helpers
```

Comment:
The `PlacementDecisionGetter` is being used by other controllers, `GetClusterChanges` and the clusters_test.go is used to test the `PlacementDecisionGetter` I think.

Reply:
I removed this because functions used in pkg/common/helpers/clusters.go are not found after I updated open-cluster-management.io/api to the latest commit in go.mod
Confirmed with @qiujian16, the `not found` issue will be fixed by https://github.com/open-cluster-management-io/addon-framework/pull/232.
I will update this PR then after the above PR is merged.

Reply:
@promid get it!